### PR TITLE
Updated build scripts with updated generation and added Xcode 10.2 changes

### PR DESCRIPTION
**Pull request by RomanTysiachnik**
@VerizonAdPlatforms/mobile-sdk-developers: Please review.

### CODE REVIEW

File: PlayerControls/PlayerControls.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -587,6 +588,7 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ALWAYS_SEARCH_USER_PATHS = NO;
+				CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED = YES;
```
Comment (AndriiMoskvin) [inline PlayerControls/PlayerControls.xcodeproj/project.pbxproj]: Make this `NO`

Comment (AndriiMoskvin) [reply PlayerControls/PlayerControls.xcodeproj/project.pbxproj]: Or instead this setting ignores localization warnings?

Comment (RomanTysiachnik) [reply PlayerControls/PlayerControls.xcodeproj/project.pbxproj]: Exactly

File: PlayerControls/PlayerControls.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -661,7 +663,7 @@
 		50582F651E65C41900B18908 /* Release */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				CODE_SIGN_IDENTITY = "iPhone Distribution";
+				CODE_SIGN_IDENTITY = "";
```
Comment (AndriiMoskvin) [inline PlayerControls/PlayerControls.xcodeproj/project.pbxproj]: Why you have changed this?

**Pull request merged**